code/modules/hydroponics/grown/nettle.dm

```diff
@@ -70,6 +70,11 @@
 	. = ..()
 	if(!proximity)
 		return
+	if (isliving(A))
+		var/mob/living/target = A
+		if (target.reagents)
+			var/inject_amount = rand(5, 10)
```

Comment:
base this off force to simulate the "leaves fall off" bit. Less Leaves = less reagent transferred?

This also needs to be checked after the `force > 0` on L77